src/xref2/tools.ml

```diff
@@ -1368,16 +1320,21 @@ and handle_canonical_module_real env p2 =
 
```

Comment:
The comment immediately above here needs updating as it still mentions ~add_canonical